src/main/java/org/assertj/core/api/AbstractZonedDateTimeAssert.java

```diff
@@ -414,10 +414,14 @@ public SELF isEqualToIgnoringHours(ZonedDateTime other) {
    *
    * @param expected the given value to compare the actual value to.
    * @return {@code this} assertion object.
+   * @throws AssertionError if the actual {@code ZonedDateTime} is {@code null}.
+   * @throws IllegalArgumentException if other {@code ZonedDateTime} is {@code null}.
    * @throws AssertionError if the actual {@code ZonedDateTime} is not equal to the {@link ZonedDateTime} in the actual
    *           ZonedDateTime's java.time.ZoneId.
    */
   public SELF isEqualTo(ZonedDateTime expected) {
+    Objects.instance().assertNotNull(info, actual);
+    assertDateTimeParameterIsNotNull(expected);
```

Comment:
to be consistent with `AbstractAssert.isEqualTo`, this assertion should pass when both actual and expected are null.

Reply:
Right! I will work on this. Thanks for review and suggestion.